documents/registernach.md

```diff
@@ -130,7 +141,14 @@ $api->subscription->createSubscriptionRegistration(array('customer'=>array('name
 | amount*   | integer      | The amount to be captured (should be equal to the authorized amount, in paise) |
 | currency*   | string  | The currency of the payment (defaults to INR)  |
 | description*  | string      | A brief description of the payment.   |
-| subscription_registration   | array  | All keys listed [here](https://razorpay.com/docs/api/recurring-payments/paper-nach/auto-debit/#121-create-a-registration-link) are supported  |
+| subscription_registration["first_payment_amount"]      | integer  | The amount, in paise, the customer should be auto-charged in addition to the authorization amount. |
+| subscription_registration["auth_type"]           | string  | Possible values `nach`|
+| subscription_registration["max_amount"]           | integer  | The maximum amount, in paise, that a customer can be charged in one transaction.|
```

Comment:
-         All keys listed here are supported for customer

Reply:
added list link like before